js/ui/tooltips/number-scrubber.js

```diff
@@ -1,5 +1,10 @@
-// A description of general tooltip flow can be found in tooltip-engine.js
-TooltipEngine.classes.numberScrubber = TooltipBase.extend({
+import $ from "jquery";
+import "webpack-jquery-ui/draggable";
```

Comment:
I found a library that claimed to make it easy to bring in jquery-ui. It seems to work.